docs/.vitepress/config.ts

```diff
@@ -106,10 +106,10 @@ export default defineConfig({
         ]
       },
       {
-        text: 'v3 (next)',
+        text: 'v3 (stable)',
         items: [
           {
-            text: 'v2.x (stable)',
+            text: 'v2 (legacy)',
```

Comment:
This is the text in https://vuejs.org/, I think we can use the same
```suggestion
            text: 'Vite 2 Docs',
```

Reply:
![image](https://user-images.githubusercontent.com/49056869/178718722-81d6328c-a684-4b46-aa00-9260033990da.png)
I feel it difficult to recognize this as a version toggle. Maybe I'm just used to the original one.

Reply:
![image](https://user-images.githubusercontent.com/49056869/178722284-c58e73b1-2671-4f10-bb50-21b3ad6c9d37.png)
Looks good to me 👍